packages/devtools/client/composables/state-tabs.ts

```diff
@@ -95,6 +95,9 @@ export function useEnabledTabs() {
   return computed(() => tabs.value
     .filter((tab, idx) => {
       const _tab = tab as ModuleBuiltinTab
+      if (settings.pinnedTabs.value.includes(_tab.name))
+        _tab.category = 'pinned'
```

Comment:
I guess we should not mutate the object, and not do it in a `.filter()`

Reply:
Indeed, I will try to fix the problem without modifying the category.